tests/test-sources/plugins/utils/flash.nix

```
@@ -0,0 +1,203 @@
+{
+  empty = {
+    plugins.flash.enable = true;
+  };
+
+  defaults = {
+    plugins.flash = {
+      enable = true;
+
+      labels = "asdfghjklqwertyuiopzxcvbnm";
+
+      search = {
```

```suggestion
      labels = "asdfghjklqwertyuiopzxcvbnm";
      search = {
```

Could you remove the blank-lines between attrs please ?
-> I hove nothing smarter to say about this PR X)